alcs-frontend/src/app/services/compliance-and-enforcement/compliance-and-enforcement.service.ts

```diff
@@ -12,8 +12,15 @@ export enum Status {
 export interface FetchOptions {
   withSubmitters?: boolean;
   withProperty?: boolean;
+  withAssignee?: boolean;
 }
 
+export const DEFAULT_C_AND_E_FETCH_OPTIONS: FetchOptions = {
+  withSubmitters: true,
+  withProperty: true,
+  withAssignee: true,
+};
+
```

Comment:
I noticed that every call to fetch a C&E file was using the same options (so far), and was necessitating updates in several places each time a new relation was added, so I created these default options.